cblast/features.py

```diff
@@ -581,12 +581,22 @@ def compute_connection_similarity_features(npclient, neuronlist,
         for iter1 in range(0, len(neuronlist), 100):
             currlist = neuronlist[iter1:iter1+100]
             
-            outputsquery=f"WITH {currlist} AS TARGETS MATCH(n :Neuron)-[x :ConnectsTo]->(m :Neuron) WHERE n.bodyId in TARGETS AND m.status=\"Traced\" AND x.weight >= {minconn} RETURN n.bodyId AS body1, x.roiInfo AS info, m.bodyId AS body2, m.type AS type, x.weight AS weight ORDER BY weight DESC"
-
-            inputsquery=f"WITH {currlist} AS TARGETS MATCH(n :Neuron)<-[x :ConnectsTo]-(m :Neuron) WHERE n.bodyId in TARGETS AND m.status=\"Traced\" AND x.weight >= {minconn} RETURN n.bodyId AS body1, x.roiInfo AS info, m.bodyId AS body2, m.type AS type, x.weight AS weight ORDER BY weight DESC"
+            outputsquery=f"""\
+                WITH {currlist} AS TARGETS 
+                MATCH(n :Neuron)-[x :ConnectsTo]->(m :Neuron) 
+                WHERE n.bodyId in TARGETS AND m.status IN {statuses} AND x.weight >= {minconn} 
+                RETURN n.bodyId AS body1, x.roiInfo AS info, m.bodyId AS body2, m.type AS type, x.weight AS weight 
+                ORDER BY weight DESC
+                """
+
+            inputsquery=f"""\
+                WITH {currlist} AS TARGETS 
+                MATCH(n :Neuron)<-[x :ConnectsTo]-(m :Neuron) 
+                WHERE n.bodyId in TARGETS AND m.status IN {statuses} AND x.weight >= {minconn} 
+                RETURN n.bodyId AS body1, x.roiInfo AS info, m.bodyId AS body2, m.type AS type, x.weight AS weight 
+                ORDER BY weight DESC
+                """
          
-            print(f"fetch batch {iter1}")
```

Comment:
Good to remove this.  Should a progress bar be added instead?

Reply:
I added that in the "think about" list. It's not clear to me which of these functions are meant to be building blocks (which should maybe take a progress callback) and which are meant to be top-level, end-user functions (which should provide the progress bar). So I put it off, reasoning that we'd figure it out as we are working with it more now.